server/routes/induction/common/wantToAddQualificationsController.ts

```diff
@@ -19,6 +19,9 @@ export default abstract class WantToAddQualificationsController extends Inductio
     next: NextFunction,
   ): Promise<void> => {
     const { prisonerSummary, prisonerFunctionalSkills } = req.session
+    const { prisonNumber } = req.params
+    this.addCurrentPageToHistory(req, `/prisoners/${prisonNumber}/induction/want-to-add-qualifications`)
```

Comment:
I'm assuming this is not in an `if` block as per the other controllers because this page (Do they want to add qualifications?) is only accessible as part of changing the induction from long to short , and that you cannot access this question/page via a "Change" link on the main Education & Training or Work & Interests pages?